spanner/spanner_samples.rb

```diff
@@ -1873,6 +1873,32 @@ def set_custom_timeout_and_retry project_id:, instance_id:, database_id:
   # [END spanner_set_custom_timeout_and_retry]
 end
 
+def commit_stats project_id:, instance_id:, database_id:
+  # [START spanner_get_commit_stats]
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+  # database_id = "Your Spanner database ID"
+
+  require "google/cloud/spanner"
+
+  spanner = Google::Cloud::Spanner.new project: project_id
+  client  = spanner.client instance_id, database_id
+
+  records = [
+    { SingerId: 1, FirstName: "Marc",     LastName: "Richards" },
+    { SingerId: 2, FirstName: "Catalina", LastName: "Smith"    }
+  ]
+  commit_options = { return_commit_stats: true }
+  resp = client.upsert "Singers", records, commit_options: commit_options
```

Comment:
Sorry that I didn't notice this before. The reference sample we did had the following logic (Node.js):

```
    const [response] = await albumsTable.upsert(
      [
        {SingerId: '1', AlbumId: '1', MarketingBudget: '200000'},
        {SingerId: '2', AlbumId: '2', MarketingBudget: '400000'},
      ],
      {returnCommitStats: true}
    );
```

It would be better if we could keep the samples consistent. Let me know if there's some conflict with the tests that I didn't foresee.

Reply:
updated example.